Sources/AppcuesKit/Presentation/Debugger/UIDebugger.swift

```diff
@@ -148,14 +150,19 @@ extension UIDebugger: DebugViewDelegate {
         }
     }
 
-    private func captureScreen(authorization: Authorization) {
+    private func captureScreen(authorization: Authorization, dismissed: Bool = false) {
```

Comment:
just curious why the `dismissed` param is needed, if the guard is already checking the `experienceRenderer.getCurrentStepIndex() == nil`, which would get set to nil after the `dismissCurrentExperience` completes?

Reply:
Good question. I started with the flag and then added the current step check afterwards to skip the dismiss call if it's unneeded, but it didn't cross my mind that the flag became redundant. I'll update it!